src/assets/locales/es.json

```diff
@@ -173,288 +173,288 @@
-  "preferences.category_touchscreen": "Pantalla táctil",
+  "preferences.category_touchscreen": "Pantallas táctiles",
   "preferences.preferredCamera": "Cámara preferida",
-  "preferences.muteMicOnEntry": "Silenciar micrófono al entrar",
-  "preferences.globalVoiceVolume": "Volumen de voz entrante",
-  "preferences.globalMediaVolume": "Volumen de medios",
-  "preferences.disableSoundEffects": "Desactivar efectos de sonido",
-  "preferences.disableEchoCancellation": "Desactivar cancelación de eco del micrófono",
+  "preferences.muteMicOnEntry": "Silenciar el micrófono al entrar",
+  "preferences.globalVoiceVolume": "Volumen de la voz entrante",
+  "preferences.globalMediaVolume": "Volumen de multimedios",
+  "preferences.disableSoundEffects": "Desactivar efectos sonoros",
+  "preferences.disableEchoCancellation": "Desactivar la anulación del eco del micrófono",
   "preferences.disableNoiseSuppression": "Desactivar la supresión de ruido del micrófono",
   "preferences.disableAutoGainControl": "Desactivar el control automático de ganancia del micrófono",
-  "preferences.snapRotationDegrees": "Rotación por pasos (en grados)",
+  "preferences.snapRotationDegrees": "Rotación por bloque (en grados)",
   "preferences.disableMovement": "Deshabilitar movimiento",
-  "preferences.disableBackwardsMovement": "Desactivar el movimiento hacia atrás",
-  "preferences.disableStrafing": "Desactivar movimiento lateral",
-  "preferences.disableTeleporter": "Desactivar teletransportador",
-  "preferences.movementSpeedModifier": "Modificador de velocidad de movimiento",
-  "preferences.enableOnScreenJoystickLeft": "Habilitar el joystick izquierdo en pantalla para moverse",
-  "preferences.enableOnScreenJoystickRight": "Habilita el joystick derecho en pantalla para mirar alrededor",
+  "preferences.disableBackwardsMovement": "Deshabilitar movimiento hacia atrás",
+  "preferences.disableStrafing": "Deshabilitar desplazamiento lateral",
+  "preferences.disableTeleporter": "Deshabilitar teletransportación",
+  "preferences.movementSpeedModifier": "Modificador de la velocidad de movimiento",
+  "preferences.enableOnScreenJoystickLeft": "Habilitar el controlador para caminar, a la izquierda de la pantalla",
+  "preferences.enableOnScreenJoystickRight": "Habilitar el controlador para girar, a la derecha de la pantalla",
   "preferences.invertTouchscreenCameraMove": "Invertir la dirección del movimiento de la cámara para pantallas táctiles",
-  "preferences.browserDefault": "Navegador predeterminado",
+  "preferences.browserDefault": "Navegador por defecto",
   "preferences.locale": "Idioma",
-  "preferences.onlyShowNametagsInFreeze": "Mostrar etiquetas de nombre solo mientras estás congelado",
-  "preferences.animateWaypointTransitions": "Animar transiciones de puntos de referencia",
+  "preferences.onlyShowNametagsInFreeze": "Solo mostrar las etiquetas de nombre en modo contextual",
+  "preferences.animateWaypointTransitions": "Animar transición de trayectoria",
```

Comment:
"Animar transición de trayectoria" should be "Animar transiciones de puntos de referencia"